### DISCUSSION TITLE
Can not sign up with an email address

### DISCUSSION THREAD

**Discussion by jszwedko** [artifacthub/hub#4173] [Q&A] [answered]
Hi all!

I'm trying to sign up via email but I keep receiving:

> An error occurred registering the user, please try again later.

I've tried multiple email addresses. Looking at the browser console I seem to be getting a 500 for the POST to https://artifacthub.io/api/v1/users. Does email sign up currently not work?

Thank you!

**Comment by tegioz** [comment, marked as answer]
Hi @jszwedko 👋

We've just tested it and it worked fine for us.

Let us take a look at the logs to try to find out why your attempt failed. Can you share the approximate time you did it?

Thanks!

**Reply by jszwedko** [reply]
Thanks for looking into this! I tried again just now and got the same error (18:13 UTC).

**Reply by tegioz** [reply]
Hi!

The email provided is already used by another account. I've seen some more attempts coming from the same IP at around 16:50 UTC, all returning the same error. Maybe you can try recovering the password for the account using that email from the sign in modal.

Hope this helps 🙂

**Reply by jszwedko** [reply]
Aha, thank you! That worked. I guess both emails I tried were already in-use. I was able to reset the password. A small suggestion would be to return that as the error message ("email already in use") since the current error is pretty vague.

**Reply by tegioz** [reply]
No worries! Thanks, we'll consider it 🙂
